### PR TITLE
language formatting

### PR DESCRIPTION
**Pull request by tarun3kumar**
First language formatting attempt discussed on - https://github.com/SeleniumHQ/docs/issues/69
Not an expert of all the languages hence they should be proof read.

**Updates:** I am having a second though about writing code samples in html. Working on old selenium do in rst was easier as it did not require any special encoding (for <, > etc). But this is something one needs to take care of when writing code sample in html. What do other doc authors think of it?

### CODE REVIEW

File: start.html [hunk 1]
```diff
@@ -99,7 +99,49 @@
...
-<pre><code class=java>WebElement cheese = driver.findElement(By.id("cheese"));</code></pre>
+    <div class="tabs">
+        <ul class="tabs">
+            <li><a href="#java">java</a></li>
+            <li><a href="#ruby">ruby</a></li>
+            <li><a href="#python">python</a></li>
+            <li><a href="#c-sharp">c#</a></li>
+            <li><a href="#perl">perl</a></li>
+            <li><a href="#php">php</a></li>
+            <li><a href="#jscript">javascript</a></li>
```
Comment (andreastt) [inline start.html:110]: I wonder if we can use a customa ttribute `data-lang` to define the language. That seems more appropriate than hijacking anchor names.

File: start.html [hunk 2]
```diff
@@ -99,7 +99,49 @@
 how to find elements on the page. WebDriver offers a number of built-in selector
 types, amongst them finding an element by its ID attribute:
 
-<pre><code class=java>WebElement cheese = driver.findElement(By.id("cheese"));</code></pre>
+    <div class="tabs">
+        <ul class="tabs">
```
Comment (andreastt) [inline start.html:103]: Drop outer `<div>` and rename from “tabs” to maybe “languages”?

On a more general note I think this list can automatically be generated and inserted with JavaScript in the future.

File: start.html [hunk 3]
```diff
@@ -99,7 +99,49 @@
...
+            <li><a href="#python">python</a></li>
+            <li><a href="#c-sharp">c#</a></li>
+            <li><a href="#perl">perl</a></li>
+            <li><a href="#php">php</a></li>
+            <li><a href="#jscript">javascript</a></li>
+        </ul>
+
+        <div class="tab" id="ruby">
+            <pre><code class=java>element = driver.find_element(:id, 'cheese')</code></pre>
+        </div>
```
Comment (andreastt) [inline start.html:115]: Here again drop the outer `<div>`.  We can identify the `<pre>` block to show by finding the immediate descendant from the language list/“tabs” above.

**Comment by tarun3kumar** [comment]
I closed the pull request as not comfortable with whole idea of tackling code examples in html, getting rid of encoding etc.

**Pull request closed without merging**